agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -1369,56 +1369,56 @@ func (dg *dockerGoClient) Stats(ctx context.Context, id string, inactivityTimeou
 		}()
```

Comment:
The changes here are because functional tests caught that polling metrics was taking longer to populate the task stats API versus streaming stats.

So change the behavior to poll for docker stats immediately on startup. This is not ideal from a load perspective but is a necessity to avoid unintentional behavior changes.